pkg/metrics/ddl.go

```diff
@@ -86,6 +95,10 @@ var (
 	AddIndexScanRate      *prometheus.HistogramVec
 )
 
+func init() {
+	registeredJob = make(map[int64]*metric.Common, 64)
```

Comment:
Could you add a check to `CheckIngestLeakageForTest` to ensure the metrics are unregistered?

Reply:
[`4aefb38` (#60904)](https://github.com/pingcap/tidb/pull/60904/commits/4aefb38bab3fd7c3a4d8475b3818527e5ac0a55c)